### ISSUE TITLE
Include module revision tests derived from

### ISSUE THREAD

**Issue by rasamassen** [LeaVerou/css3test#14]
The modules on the CSS3Test are rather stable, but sometimes minor or even major changes occur with a new working draft. Whether just in the code or on the CSS3Test page, it would be helpful to display which module revision the tests were derived from, if for no other reason that providing a means to track when tests might be out-of-date and the latest module revision needs to be consulted to determine if changes are needed.

Basically, something somewhere that says, in not so many words, "Backgrounds tests based on DEV module, revision date 15 February 2011." Or "Background tests based on TR module, revision date 14 February 2012." If they were based on the DEV module, this simple line would indicate that the tests are out-of-date and need to be revised, as a new background DEV module is out.
